### ISSUE TITLE
Path completion upwards, starting with ".." [TAB] does not complete to "../"

### ISSUE THREAD

**Issue by amanessinger** [warpdotdev/Warp#1453]
### Describe the solution you'd like?

```text
In bash, navigating upwards, I can type ".." followed by [TAB]. This completes to "../". 
The next [TAB] just rings the bell, and another [TAB] offers suggestions.

ImO, Warp should complete ".." upon first [TAB] to "../" and immediately, without another [TAB], 
show the completions menu, starting with "..", followed by subdirectories of "../" ($CWD excluded), 
and then (if it's at the start of the commandline) all executables in "../". 

A hyper-luxury solution would make order (execs or dirs first) configurable.
```

### Additional context

The current behavior might not be a big thing for users of US keyboards, where you can type "/" easily. On my German keyboard, I need to press [SHIFT]+7, which is really awkward. Most non-US keyboards are similar.

**Comment by elviskahoro** [comment]
This would be great for ergonomics. Thanks for reporting!

**Comment by szgupta** [comment]
This was fixed in `v0.2022.08.16.10.16.stable_02`!

**Comment by szgupta** [comment]
Closing out for now -- feel free to re-open if this still doesn't work the way you expect it to
